openshift_cli_installer/manifests/clusters.example.yaml

```diff
@@ -104,3 +104,4 @@ clusters:
   worker_replicas: 2
   worker_flavor: m5.4xlarge
   worker_root_disk_size: 128
+  log-level: info # optional, default: "info", supported options are debug, info, warn, error
```

Comment:
We need to check if the level that the user send is one of the supported one (In `UserInput` code)

Reply:
I don't see the check. did you push it?